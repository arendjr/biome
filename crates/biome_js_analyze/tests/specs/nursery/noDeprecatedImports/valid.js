/* should not generate diagnostics */
import { NOT_REALLY_DEPRECATED_CONSTANT as CONSTANT } from "./utils";

import { modernUtil } from "./utils";

import { Component } from "./component";
