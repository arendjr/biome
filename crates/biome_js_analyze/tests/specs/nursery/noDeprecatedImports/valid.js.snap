---
source: crates/biome_js_analyze/tests/spec_tests.rs
expression: valid.js
---
# Input
```js
/* should not generate diagnostics */
import { NOT_REALLY_DEPRECATED_CONSTANT as CONSTANT } from "./utils";

import { modernUtil } from "./utils";

import { Component } from "./component";

```
